File: regression/java/fixtures/References.java
package com.regression;

public class References {
    public static void callThings() {
        Main.greet("Alice");
        Main.farewell("Bob");
        
        Calculator calc = new Calculator();
        calc.add(1, 2);
        calc.subtract(5, 3);
        
        Dog dog = new Dog("Rex");
        dog.speak();
    }
}

File: regression/ruby/fixtures/references.rb
require_relative "valid"

def call_greet
  greet("Alice")
end

def call_farewell
  farewell("Bob")
end

class MathStudent < Student
  def initialize(name, grade)
    super(name)
    @grade = grade
  end
end

call_greet
call_farewell
